datalad/tests/test_annexrepo.py

```diff
@@ -367,6 +367,49 @@ def test_AnnexRepo_get_file_backend(src, dst):
         assert_raises(CommandNotAvailableError, ar.migrate_backend,
                       'test-annex.dat', backend='SHA1')
 
+
+@with_tempfile
+def test_AnnexRepo_always_commit(path):
+
+    repo = AnnexRepo(path)
+    file1 = get_most_obscure_supported_name() + "_1"
+    file2 = get_most_obscure_supported_name() + "_2"
+    with open(opj(path, file1), 'w') as f:
+        f.write("First file.")
+    with open(opj(path, file2), 'w') as f:
+        f.write("Second file.")
+
+    repo.always_commit = True
```

Comment:
that should be the default one, so better make it "explicit" by making it "implicit" (i.e. not setting here) ;)